go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/99designs/keyring v1.1.6
 	github.com/armon/go-metrics v0.3.11
 	github.com/bgentry/speakeasy v0.1.0
-	github.com/btcsuite/btcd v0.22.1
```

Comment:
This doesn't bump anything. In the `provenance` repo, it caused the version to actually regress back to `v0.22.0-beta`.

I'll look deeper in to this a bit later, but for now, this PR should NOT be merged.